bindata/v3.11.0/kube-apiserver/recovery-pod.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: kube-system  # for disaster recovery we need namespace that is always present
```

Comment:
no you don't

Reply:
yeah, this will come up, I was thinking about the other phase where kubelet syncs pod status into the apiserver but we don't ever setup the credentials, so it will never get as far to spit those errors.